src/openforms/submissions/logic/actions.py

```diff
@@ -1,14 +1,19 @@
 from dataclasses import dataclass
-from typing import Any, Dict, Optional, TypedDict
+from typing import Any, Dict, Mapping, Optional, Type, TypedDict
+
+from json_logic import jsonLogic
 
 from openforms.formio.service import FormioConfigurationWrapper
 from openforms.forms.constants import LogicActionTypes
-from openforms.forms.models import FormVariable
-from openforms.typing import JSONObject
+from openforms.forms.models import FormLogic, FormVariable
+from openforms.typing import DataMapping, JSONObject
 from openforms.utils.json_logic import ComponentMeta, introspect_json_logic
+from openforms.variables.models import ServiceFetchConfiguration
 
+from ..logic.service_fetching import perform_service_fetch
```

Comment:
```suggestion
from .service_fetching import perform_service_fetch
```
I think?

Reply:
Sure... After lunch and coffee.  I just rebased onto master... Hungry now. :wink: